### DISCUSSION TITLE
Engraving sketches and portait

### DISCUSSION THREAD

**Discussion by preissasaf** [arkypita/LaserGRBL#1673] [Q&A]
Hi,
I'm new here and in engraving with laser. 
Well, years ago I've worked with laser, but two weeks ago I've bought my ORTUR OLM2 PRO S2 laser machine.
I'm trying to engrave sketches and B/W pictures, but it seems that I'm doing something wrong with the definintions of the images or withing the user interface.
I will appreciate if anyone here can guide me or send any guide lines that will help me.
Thanks all,
Asaf

**Comment by gmmanonymus111** [comment]
Please refer to [LaserGRBL's usage manual](https://lasergrbl.com/usage/raster-image-import/line-to-line-tool/) on line-to-line engraving.
See more about image engraving under `Usage` -> `Raster image import`
